lambda/tile_upload_lambda.py

```diff
@@ -29,6 +29,8 @@
 
 from botocore.exceptions import ClientError
 
+print("$$$ IN TILE UPLOAD LAMBDA $$$")
```

Comment:
This is really unrelated but I found it helpful to be able to search multilambda logs for specific types of lambas.  that is what this is giving us.